meetings/2022-06-30.md

```diff
@@ -39,38 +42,99 @@
 
 ### WIP Implementation Feedback
 * SC: We'll allocate some time for implementation feedback going forward. Links to servers/applications welcome.
+* CB: Heads up that Apple announced iOS16 support for Web Push notifications. So it'll work across all devices. Safari didn't support but on MacOS they added 1-2 years ago but... people wanted it so now they know. The community is developers was very focused and Apple gave in.
+* SC: More adoption is good sign.
+* RG: What are you mainly developing when it comes to push notifications.
+* SC: I'm not currently developing push notifications but possible because dokieli can be used as a browser extension so that might help out to indicate when there is an annotation available on a document.
+* RG: Is there a server implementation that's going hand in hand with the notifications protocol we are developing here.
+* SC: So there are private and public implementations - some are experimental and some are waiting for a more stable version of the spec. I think this will happen in the next months when the Notifications Protocol will stabilise as much as it could any way, as well as the Solid Protocol. It's ongoing work.
```

Comment:
```suggestion
* SC: There are private and public implementations. Some are experimental, and some are waiting for a more stable version of the spec. I think this will happen in the next months when the Notifications Protocol will stabilise (as much as it can, anyway), as well as the Solid Protocol. It's ongoing work.
```